modules/nvidia-gpu-virtualization-with-gpu.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * architecture/nvidia-gpu-architecture-overview.adoc
+
+:_content-type: CONCEPT
+[id="nvidia-gpu-virtualization-with-gpu_{context}"]
+= Virtualization with vGPU
+
+NVIDIA vGPU allows for direct access to a single physical GPU from virtual machines (VMs). You can create virtual GPUs that can be shared by VMs across the enterprise and accessed by other devices.
```

Comment:
```suggestion
Virtual machines (VMs) can directly access a single physical GPU using NVIDIA vGPU. You can create virtual GPUs that can be shared by VMs across the enterprise and accessed by other devices.
```
Reworded to avoid "allow" language:
"Avoid stating that inanimate objects grant abilities to people, as in “the product allows you to…”. Whenever possible, use a direct, user-focused alternative such as “you can use the product to…” or “with this product, users can…”."